extensions/api/identity-api/verifiable-credentials-api/src/main/java/org/eclipse/edc/identityhub/api/verifiablecredentials/v1/unstable/VerifiableCredentialsApi.java

```diff
@@ -55,6 +55,44 @@ public interface VerifiableCredentialsApi {
     )
     VerifiableCredentialResource getCredential(String id, SecurityContext securityContext);
 
+    @Operation(description = "Adds a new VerifiableCredential into the system.",
+            operationId = "addCredential",
+            parameters = {
+                    @Parameter(name = "participantId", description = "Base64-Url encode Participant Context ID", required = true, in = ParameterIn.PATH)
+            },
+            requestBody = @RequestBody(content = @Content(schema = @Schema(implementation = VerifiableCredentialManifest.class))),
+            responses = {
+                    @ApiResponse(responseCode = "204", description = "The VerifiableCredential was successfully created."),
+                    @ApiResponse(responseCode = "400", description = "Request body was malformed, or the request could not be processed",
+                            content = @Content(array = @ArraySchema(schema = @Schema(implementation = ApiErrorDetail.class)), mediaType = "application/json")),
+                    @ApiResponse(responseCode = "401", description = "The request could not be completed, because either the authentication was missing or was not valid.",
+                            content = @Content(array = @ArraySchema(schema = @Schema(implementation = ApiErrorDetail.class)), mediaType = "application/json")),
+                    @ApiResponse(responseCode = "404", description = "A VerifiableCredential with the given ID does not exist.",
```

Comment:
the "404" seems weird, because a VC with the given ID _shouldn't_ exist. Copy-paste :)?